src/platforms/blackpillv2/Makefile.inc

```diff
@@ -27,6 +27,16 @@ endif
 
 VPATH += platforms/stm32
 
+ifeq ($(ALTERNATIVE_PINOUT) <= 0, 1)
+CFLAGS += -DALTERNATIVE_PINOUT=0
+else ifeq ($(ALTERNATIVE_PINOUT), 1)
+CFLAGS += -DALTERNATIVE_PINOUT=1
+else ifeq ($(ALTERNATIVE_PINOUT), 2)
```

Comment:
Given there's only one alternative pinout currently defined, and you're mapping `$(ALTERNATIVE_PINOUT)` to `CFLAGS`' `-DALTERNATIVE_PINOUT=` values 1:1, it seems to us to make sense to pass the value through directly as `CFLAGS += -DALTERNATIVE_PINOUT=$(ALTERNATIVE_PINOUT)`

Reply:
Updated in the way you suggested.

I have also added `ifndef ALTERNATIVE_PINOUT` such that the code can still be built without passing the `ALTERNATIVE_PINOUT` to make.
This was already possible using the implementation you commented on, but was not possible without adding the `ifndef` line.